packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -200,47 +200,50 @@ test.serial('writes account addresses to vstorage', async t => {
   await documentStorageSchema(t, storage, doc);
 });
 
-test.serial('makes usdc advance', async t => {
-  const {
-    walletFactoryDriver: wd,
-    storage,
-    agoricNamesRemotes,
-    harness,
-  } = t.context;
-  const oracles = await Promise.all([
-    wd.provideSmartWallet('agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8'),
-    wd.provideSmartWallet('agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr'),
-    wd.provideSmartWallet('agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj'),
-  ]);
-  await Promise.all(
-    oracles.map(wallet =>
-      wallet.sendOffer({
-        id: 'claim-oracle-invitation',
-        invitationSpec: {
-          source: 'purse',
-          instance: agoricNamesRemotes.instance.fastUsdc,
-          description: 'oracle operator invitation',
-        },
-        proposal: {},
-      }),
-    ),
+test.serial('LP deposits', async t => {
+  const { walletFactoryDriver: wd, agoricNamesRemotes } = t.context;
+  const lp = await wd.provideSmartWallet(
+    'agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8',
   );
 
-  const lp = oracles[0]; // somewhat arbitrary
-
-  // @ts-expect-error it doesnt recognize usdc as a Brand type
+  // @ts-expect-error it doesnt recognize USDC as a Brand type
   const usdc = agoricNamesRemotes.vbankAsset.USDC.brand as Brand<'nat'>;
+  // @ts-expect-error it doesnt recognize FastLP as a Brand type
+  const fastLP = agoricNamesRemotes.vbankAsset.FastLP.brand as Brand<'nat'>;
+
+  // Send a bad proposal first to make sure it's recoverable.
+  // (https://github.com/Agoric/agoric-private/issues/234)
```

Comment:
```suggestion
```

let's avoid pointers from public stuff to private stuff

Reply:
Done